File: tools/allEven.py
def allEven(n):
    
    even = True

    while (n > 0) and (even == True):
        if (n % 10) % 2 == 1 :
            even = False
        n = n // 10
        print(n)
    return even
print(allEven(345))
print(allEven(236))

File: tools/allEven.js
function allEven (n) {
    var even = true;
    while (n > 0 && even == true) {
        if ((n % 10) % 2 == 1) {
            even = false
        }
        n = Math.floor(n/10)
        console.log(n)
        return even
    }
}
console.log(allEven (345))
console.log(allEven (236))
